doc/source/ray-core/ray-dashboard.rst

```diff
@@ -274,6 +274,58 @@ You get the same information from the :ref:`Ray state APIs <state-api-overview-r
 
 You can expand the table to see a list of each task, actor, and placement group.
 
+.. _dash-serve-view:
+
+Serve View
+----------
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/serve.png
+    :align: center
+
+The Serve view lets you monitor the status of your :ref:`Ray Serve <rayserve>` applications. A Serve application can be deployed using the `serve deploy` CLI command.
+
+The initial page showcases your general Serve configurations, a list of all your Serve applications, and, if you have :ref:`metrics <ray-metrics>` configured, some high-level
```

Comment:
why is this "if" you have metrics configured? metrics should work out of the box right?

or is this about setting up grafana on the head node? maybe we can make the wording a little more precise here.